sections/futureWork.tex

```diff
@@ -0,0 +1,9 @@
+\section{Future Work}\label{sec:future_work}
+Due to the promising results provided by the modified pachinko model, investigating this model further might be beneficial to incorporating meta-data into topic models.
+However, testing these model on multiple datasets, needed to be accounted for, since the generalizability of these models is not explored within this paper.

```

Comment:
```suggestion
However, testing these models on multiple datasets needs to be accounted for since the generalizability of these models is not explored within this paper.
```